indico/modules/rb/controllers/user/reservations.py

```diff
@@ -457,9 +457,30 @@ def _checkParams(self):
     def _get_view(self, **kwargs):
         return RHRoomBookingNewBookingSimple._get_view(self, clone_booking=self._reservation, **kwargs)
 
+    def _update_datetime(self):
+        """Make necessary changes to reservation's start and end datetime.
+
+        Move the start date to the current day, adjust the end date
+        accordingly and if the start datetime is still in the past, change the
+        start and end time to the current (rounded up) time.
+
+        :return: A tuple with the new start_dt, end_dt
```

Comment:
"times", rly? One time, two times? :stuck_out_tongue: 
Anyway, it's datetime, the date is also altered.